xkcd/pkg/index/index.go

```diff
@@ -0,0 +1,158 @@
+package index
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/vespian/go-exercises/xkcd/pkg/pbuff"
+	"github.com/vespian/go-exercises/xkcd/pkg/types"
+	"github.com/vespian/go-exercises/xkcd/pkg/web"
+)
+
+func serialize(a types.AllStories, t types.OndiskSerialization,
+) (
+	blob []byte,
```

Comment:
Code that speaks...to whom? The reader of the code is presumably someone familiar with golang and golang style and naming conventions - who will see "buf []byte" and immediately understand.